docs/docs/configuration/systask.md

```diff
@@ -552,3 +552,51 @@ Task Input payload sample
 ```
 
 The task is marked as ```FAILED``` if the message could not be published to the Kafka queue. 
+
+
+## Do While Task
+
```

Comment:
It could be useful to add information about the output as well, and eventually about how to reference looped tasks inside another task.

Reply:
Sure. will add up here.

Reply:
@andrea11 Updated the doc. Please let me know if you find it still unclear how to use do while task.

Reply:
Can you also call out the limitation here? From this [comment](https://github.com/Netflix/conductor/pull/1169#issuecomment-539589612).